src/schema/schema.graphql

```diff
@@ -2731,6 +2742,8 @@ type Query {
 	taskFolders: [Folder]
 
 	getTag: [Tag]
+
+	getNewMail: NewMail
```

Comment:
Shouldn't it be an array, what if more then one emails arrive at the same time

Reply:
Yes, that should have been an array, changed. Also changed related logic.